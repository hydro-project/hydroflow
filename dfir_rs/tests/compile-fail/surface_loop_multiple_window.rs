fn main() {
    let mut df = dfir_rs::dfir_syntax! {
        a = source_iter(0..10);
        loop {
            loop {
                a -> batch() -> null();
            }
        }
    };
    df.run_available();
}
