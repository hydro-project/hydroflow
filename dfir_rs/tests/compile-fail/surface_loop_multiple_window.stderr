error: Operator input edge may not cross multiple loop contexts.
 --> tests/compile-fail/surface_loop_multiple_window.rs:6:22
  |
6 |                 a -> batch() -> null();
  |                      ^^^^^^^
